src/components/EntryForm.js

```diff
@@ -0,0 +1,31 @@
+//import {useState} from 'react';
+
+import useVacationStore from './hooks/useVacationStore';
+// import Vacations from './vacations';
+
+export default function EntryForm() {
+	// 	const [entries, setEntries] = useState('');
+
+	return (
+		<>
+			<form>
+				<label htmlFor="chooseVacation">Choose a vacation: </label>
+				<select name="vacations" id="vacation-select">
+					{useVacationStore.vacations.map(vacation => (
+						<option
+							key="vacationSelect"
+							value={useVacationStore.vacations.vacationName}
+							required
+						>
+							{vacation.vacationName}
+						</option>
+					))}
+				</select>
+				<input type="date" name="date" required />
+				<label>Diary entry: </label>
+				<textarea name="entry" placeholder="Today I ..."></textarea>
```

Comment:
The `input` element needs a label element. And the `label` element and ´textarea` element are not connected with `htmlFor` and `id`. 👀